functions/api/category.js

```diff
@@ -213,6 +213,7 @@ router.put('/', isAdminOrSuperAdmin, async (request, response, next) => {
     const oldData = category.data()
     let newData = request.body
     delete newData[constants.ID]
+    newData[constants.NAME] = newData[constants.NAME].toLocaleLowerCase()
```

Comment:
what are these changes for?

Reply:
Just to make sure I save names in lowercase in case of PUT operation as well as we get all the data from UI